app/views/glossary_terms/edit.html.erb

```diff
@@ -16,6 +16,5 @@
 
 <div class="form-max-width mt-4">
   <%= tag.h1 @title.humanize, class: "display-1" %>
-  <%= render partial: "form",
-             locals: { button_name: :edit_glossary_term_save, action: :edit } %>
+  <%= render "form", button_name: :edit_glossary_term_save %>
```

Comment:
In contrast to the `new` form a block is not needed here because `edit` will ignore the form when the `form` partial yields to `edit`.